### DISCUSSION TITLE
%E5%8F%98%E8%BA%AB%E5%B0%8F%E8%AF%B4%EF%BC%88%E6%9C%AA%E5%88%86%E7%B1%BB%EF%BC%89/%E6%9C%B1%E9%A2%9C%E8%A1%80%EF%BC%88%E9%9B%AA%E8%8A%8D%EF%BC%89_page/

### DISCUSSION THREAD

**Discussion by giscus** [project-polymorph/trans-digital-cn#5] [Announcements]
# %E5%8F%98%E8%BA%AB%E5%B0%8F%E8%AF%B4%EF%BC%88%E6%9C%AA%E5%88%86%E7%B1%BB%EF%BC%89/%E6%9C%B1%E9%A2%9C%E8%A1%80%EF%BC%88%E9%9B%AA%E8%8A%8D%EF%BC%89_page/

None

https://snovel.transchinese.org/%E5%8F%98%E8%BA%AB%E5%B0%8F%E8%AF%B4%EF%BC%88%E6%9C%AA%E5%88%86%E7%B1%BB%EF%BC%89/%E6%9C%B1%E9%A2%9C%E8%A1%80%EF%BC%88%E9%9B%AA%E8%8A%8D%EF%BC%89_page/?h=%E6%9C%B1

**Comment by yunwei37** [comment]
测试（这本据说挺有名的
